jboss-marshalling/src/test/java/org/infinispan/jboss/marshalling/dataconversion/DataConversionTest.java

```diff
@@ -8,24 +9,20 @@
 
 import org.infinispan.AdvancedCache;
 import org.infinispan.Cache;
+import org.infinispan.commons.dataconversion.MediaType;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
-import org.infinispan.factories.GlobalComponentRegistry;
 import org.infinispan.jboss.marshalling.commons.GenericJBossMarshaller;
-import org.infinispan.marshall.core.EncoderRegistry;
 import org.infinispan.test.CacheManagerCallable;
 import org.infinispan.test.data.Person;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "marshall.jboss.DataConversionTest")
 public class DataConversionTest extends org.infinispan.dataconversion.DataConversionTest {
+   private final GenericJBossMarshaller marshaller = new GenericJBossMarshaller();
 
    @Test
    public void testObjectEncoder() {
-      GenericJbossMarshallerEncoder encoder = new GenericJbossMarshallerEncoder(org.infinispan.dataconversion.DataConversionTest.class.getClassLoader());
-      withCacheManager(new CacheManagerCallable(
-            createCacheManager(new ConfigurationBuilder())) {
-
-         GenericJBossMarshaller marshaller = new GenericJBossMarshaller();
+      withCacheManager(new CacheManagerCallable(createCacheManager(new ConfigurationBuilder())) {
```

Comment:
This method is supposed to test the encoder itself, so I think it shouldn't change for now.